tests/test_hdate_api.py

```diff
@@ -220,3 +220,20 @@ def test_zmanim_localized_datetime(self) -> None:
             date=datetime(2019, 4, 21, 20, 30, tzinfo=_timezone), location=coord
         )
         assert not zman.issur_melacha_in_effect
+
+    def test_candle_lighting_localized_datetime(
+        self, capsys: CaptureFixture[str]
+    ) -> None:
+        """Test for issur melacha if datetime is localized."""
```

Comment:
Please rename the method and the docstring to be more appropriate so we know what we're testing:
test_candle_lighting_erev_shabbat_is_yom_tov

Test for candle lighting when erev shabbat is Yom tov should be the regular candle lighting time.

Reply:
Done